x/ibc/05-port/types/module.go

```diff
@@ -0,0 +1,77 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/cosmos-sdk/x/capability"
+	channelexported "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/exported"
+	channeltypes "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types"
+)
+
+type IBCModule interface {
```

Comment:
Placed inside `05-port/types` since that made most sense to me. Open to moving it